<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Switching Control Panel</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="input_group">
      <property name="focusPolicy">
       <enum>Qt::TabFocus</enum>
      </property>
      <property name="title">
       <string>Inputs</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="pulse_type_combobox">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="currentText">
          <string>Pulse Voltage</string>
         </property>
         <item>
          <property name="text">
           <string>Pulse Voltage</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pulse Current</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="pulse_magnitude_box">
         <property name="text">
          <string>5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="pulse_magnitude_units_label">
         <property name="text">
          <string> V</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="pulse_width_label">
         <property name="text">
          <string>Pulse Width</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pulse_width_box">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="pulse_width_units_label">
         <property name="text">
          <string>ms</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="probe_current_label">
         <property name="text">
          <string>Probe Current</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="probe_current_box">
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QLabel" name="probe_current_units_label">
         <property name="text">
          <string>uA</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="measurement_count_label">
         <property name="text">
          <string>N Per Loop</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="measurement_count_box">
         <property name="text">
          <string>200</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="loop_count_label">
         <property name="text">
          <string>Loop Count</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="loop_count_box">
         <property name="text">
          <string>2</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="rhs_layout">
      <item>
       <widget class="QGroupBox" name="ports_group">
        <property name="title">
         <string>Ports</string>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="1" column="0">
          <widget class="QLabel" name="sb_port_label">
           <property name="text">
            <string>Switch Box Port</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="sb_port_box">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="bb_port_label">
           <property name="text">
            <string>Balance Box Port</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="bb_port_box">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="dmm_port_label">
           <property name="text">
            <string>Keithley 2000 Port</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="dmm_port_box">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="button_group">
        <item>
         <widget class="QPushButton" name="start_button">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_button">
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
